ember/package.json

```diff
@@ -67,11 +69,12 @@
     "ember-template-lint": "^5.3.0",
     "ember-truth-helpers": "^3.1.1",
     "ember-uikit": "^7.0.0",
-    "eslint": "^7.0.0",
-    "eslint-config-prettier": "^8.6.0",
-    "eslint-plugin-ember": "^11.4.6",
+    "eslint": "^8.38.0",
+    "eslint-config-prettier": "^8.8.0",
+    "eslint-plugin-ember": "^11.5.2",
     "eslint-plugin-import": "^2.27.5",
     "eslint-plugin-node": "^11.1.0",
```

Comment:
You can drop `eslint-plugin-node` as it was replaced by `eslint-plugin-n`

Reply:
Done here: https://github.com/adfinis/Outdated/pull/33/commits/161c2d98d8d4c6cd5ec9e469768af2fcc8ec4ed9